<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>633</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="Enviar">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QFrame" name="frame_2">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="4" column="0" colspan="3">
               <widget class="QLineEdit" name="text_assunto">
                <property name="text">
                 <string/>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                </property>
               </widget>
              </item>
              <item row="6" column="0" colspan="3">
               <widget class="QTextEdit" name="text_corpo">
                <property name="html">
                 <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="Destinatario">
                <property name="text">
                 <string>Destinatário:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="Endereco">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>Endereço de Email:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="text_endereco">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="text_destinatario">
                <property name="text">
                 <string/>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="Corpo">
                <property name="styleSheet">
                 <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
                </property>
                <property name="text">
                 <string>Corpo:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="Assunto">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Assunto:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="Senha">
                <property name="text">
                 <string>Senha:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QPushButton" name="login_botao">
                <property name="text">
                 <string>Login</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="text_senha">
                <property name="text">
                 <string/>
                </property>
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <widget class="QPushButton" name="enviar_botao">
                <property name="text">
                 <string>Enviar</string>
                </property>
               </widget>
              </item>
              <item row="7" column="2">
               <widget class="QPushButton" name="deslogar_smtp">
                <property name="text">
                 <string>Deslogar</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="Checar">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QFrame" name="frame_3">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="5" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="9" column="2">
               <widget class="QPushButton" name="deslogar_imap">
                <property name="text">
                 <string>Deslogar</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="text_endereco_2">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="search_bar">
                <property name="text">
                 <string/>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                </property>
               </widget>
              </item>
              <item row="9" column="0" alignment="Qt::AlignLeft">
               <widget class="QPushButton" name="mostrar">
                <property name="text">
                 <string>Mostrar Mais</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QPushButton" name="pesquisar">
                <property name="text">
                 <string>Pesquisar</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="text_senha_2">
                <property name="text">
                 <string/>
                </property>
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="Endereco_2">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>Endereço de Email:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="Senha_2">
                <property name="text">
                 <string>Senha:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QComboBox" name="comboBox">
                <item>
                 <property name="text">
                  <string>Pesquise por Assunto</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Pesquise por Destinatario</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Pesquise por Conteudo</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QPushButton" name="login_botao_2">
                <property name="text">
                 <string>Login</string>
                </property>
               </widget>
              </item>
              <item row="8" column="0" colspan="3">
               <widget class="QTableWidget" name="tableWidget">
                <column>
                 <property name="text">
                  <string>Remetente</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Assunto</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>ID_email</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="Conteudo">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QFrame" name="frame_4">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QTextEdit" name="email_text"/>
              </item>
              <item alignment="Qt::AlignHCenter">
               <widget class="QPushButton" name="voltar">
                <property name="text">
                 <string>Voltar</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>868</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEnviar">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionEnviar"/>
    <addaction name="actionChecar"/>
   </widget>
   <addaction name="menuEnviar"/>
  </widget>
  <action name="actionEnviar">
   <property name="text">
    <string>Enviar</string>
   </property>
  </action>
  <action name="actionChecar">
   <property name="text">
    <string>Checar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
